packages/docusaurus/src/client/exports/__tests__/Translate.test.tsx

```diff
@@ -67,6 +67,7 @@ describe('<Translate>', () => {
   it('rejects when children is not a string', () => {
     expect(() =>
       renderer.create(
+        // eslint-disable-next-line @docusaurus/no-dynamic-i18n-messages
         <Translate id="foo">
           {/* @ts-expect-error: for test */}
           <span>aaa</span>
```

Comment:
I don't know if it should be reported like this 🤔 It's technically still "a literal", just not a string literal, and `no-dynamic-i18n-messages` sounds rather misleading. But it should absolutely be reported by this rule.

Reply:
If the meaning of "no-dynamic" is too broad, one idea is to rename the rule to something like `string-literal-i18n-messages`.
 https://eslint.org/docs/developer-guide/working-with-rules#rule-naming-conventions

Reply:
Sure, that does sound better!

Reply:
Addressed in 19a7f89.